### ISSUE TITLE
problem installing

### ISSUE THREAD

**Issue by lockmeister** [Integuru-AI/Integuru#1]
I cloned the repo, cd'd into Integuru directory and ran `poetry install -vvv` and got this error:
w
Installing the current project: integrationagent (0.1.0)
  - Building package integrationagent in editable mode

Warning: The current project could not be installed: No file/folder found for package integrationagent
If you do not want to install the current project use --no-root.
If you want to use Poetry only for dependency management but not for packaging, you can disable package mode by setting package-mode = false in your pyproject.toml file.
In a future version of Poetry this warning will become an error!

I'm using python 3.12.7

**Comment by alanalanlu** [comment]
Try changing in pyproject.toml `name = "integrationagent"` to `name = "integuru"` 

Lmk if this works!

**Comment by lockmeister** [comment]
will do. In the meantime, heres a summary of problems I had when installing:

I encountered multiple problems while trying to set up and run the project using the instructions provided. The main issues relate to package installation, module recognition, and the project’s structure.

Problems Faced:
Missing or Incorrect Package Installation:

After cloning the repository and running poetry install, I encountered a warning stating that the project could not be installed because it couldn't find the package integrationagent.
It seems like the project folder may not have been structured as expected, or the package name in the pyproject.toml file doesn't match the actual folder name.
Kernel Not Recognizing Poetry Virtual Environment:

When trying to run main.ipynb, Jupyter did not automatically pick up the virtual environment created by Poetry. This required manually registering the virtual environment using ipykernel to ensure the correct Python environment was used.
Module Not Found (ModuleNotFoundError):

Initially, there was a ModuleNotFoundError for integration_agent. Upon inspecting the project directory, it became clear that the actual package directory is named integuru, not integration_agent.
This mismatch caused the Jupyter notebook to fail, as all the imports were referencing the wrong package name. Updating the imports to reference integuru resolved this issue.
Unclear Project Structure:

There was some confusion due to the project structure not matching the imports in the provided notebook. The folder integuru/ contains the code, but the notebook and possibly the configuration refer to integration_agent, which led to module import issues.
Steps to Reproduce:
Clone the repository: git clone https://github.com/Integuru-AI/Integuru.
Run poetry install in the project directory.
Open main.ipynb and attempt to run the cells.
Observe the missing module errors (ModuleNotFoundError) and issues with package installation.
Suggested Fixes:
Ensure the project folder name in the repository (integuru) matches the module name used in the code and pyproject.toml.
Update the installation instructions to clearly explain how to set up the Jupyter environment, including registering the Poetry virtual environment with Jupyter.
Provide clearer guidance on the expected directory structure and ensure that the imports in the provided Jupyter notebook reflect the actual project structure.
